### DISCUSSION TITLE
Maximum Force for Alamode calculation

### DISCUSSION THREAD

**Discussion by zohansyahfatomi** [ttadano/alamode#230] [General]
Dear Admin,

My name is Zohan.
I plan to calculate phonon properties. 
In the documentary, it is said that:
```
After that, please optimize the internal coordinate so that the atomic forces are negligibly small.
```
How small can the maximum atomic force get reliable results in Alamode calculation?
Thank you very much.

Best regards,
Zohan

**Comment by ttadano** [comment]
Hi,

In many cases, I optimize the atomic positions until all force components become less than 0.1 eV/Å.
